layouts/partials/header.html

```diff
@@ -13,8 +13,8 @@
         <nav class="cf">
             <div class="upper-nav right">
                 <a href="/docs/">Docs</a>
+                <a href="/community/">Community</a>
```

Comment:
Let's go with "Contribute" for the name of this page. I think @andymoe and other folks brought up good points for calling it contribute over community. Apologies for the back and forth on this one.